lib/patch-series.ts

```diff
@@ -141,9 +142,10 @@ export class PatchSeries {
                 pullRequestURL,
             };
         } else {
-            if (!await git(["rev-list",
+            if (!options.noUpdate &&   // allow reprint of submitted PRs
+                !await git(["rev-list",
                             `${metadata.headCommit}...${headCommit}`],
-                           { workDir })) {
+                            { workDir })) {
```

Comment:
I think it is this extra space that [the build is complaining about](https://dev.azure.com/gitgitgadget/git/_build/results?buildId=17643&view=logs&jobId=12f1170f-54f2-53f3-20dd-22fc7dff55f9&taskId=9c939e41-62c2-5605-5e05-fc3554afc9f5&lineStart=36&lineEnd=36&colStart=1&colEnd=83).

Reply:
Sorry, did not mean for you to have to look that up.  Forgot the push would kick off the CI test and notify you.  Fixed it.

Reply:
Just let me know when I should look at this PR again 😄